contrib/format-xml/src/test/resources/logback-test.txt

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Please remove this file, it is not used.

Reply:
Removed